src/Stratis.Bitcoin/P2P/ISelfEndpointTracker.cs

```diff
@@ -1,4 +1,8 @@
-﻿using System.Net;
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
```

Comment:
all usings except for this are redundant

Reply:
Removed.